README.md

```diff
@@ -1 +1,36 @@
-# lizardboard
\ No newline at end of file
+# Lizardboard
+
+An open source clone of [geckoboard](https://www.geckoboard.com/). Interested in contributing? Take a look at the [project page](https://github.com/GuildCrafts/lizardboard/projects) and [issues page](https://github.com/GuildCrafts/lizardboard/issues) for outstanding issues.
+
+## Contributing to Lizardboard
+
+Please read the [contribution guidelines](https://github.com/GuildCrafts/lizardboard/blob/master/CONTRIBUTING.md)
+
+## Setting up Development Environment
+
+A mongodb database named lizardboard must be created prior to starting the application for the first time.
+- Brew install mongodb
+- Yarn install
+- Run `mongod` command on one tab to open up the database server
+- Run `mongo` command on another tab to open up the MongoDB shell
+
+## Technical Stack
+
+### Back End
+- [Node.js](https://nodejs.org/en/)
+- [Express.js](http://expressjs.com/)
+- [Passport](http://passportjs.org/docs)
+- [Passport-local](https://www.npmjs.com/package/passport-local)
+
+### Database
+- [Mongodb](https://docs.mongodb.com/)
+  -[Mongoose](http://mongoosejs.com/docs/guide.html)
```

Comment:
Have we officially decided on `Mongoose`? Would love to here why we decided to use this lib.